knowledge-graph/src/test/scala/ch/datascience/knowledgegraph/datasets/DatasetsGenerators.scala

```diff
@@ -41,10 +41,20 @@ object DatasetsGenerators {
       maybeUrl         <- Gen.option(datasetUrls)
       sameas           <- sameAs
       maybeDescription <- Gen.option(datasetDescriptions)
+      keywords         <- nonEmptyList(datasetKeywords)
```

Comment:
I think keywords could potentially be empty so maybe generator should generate such cases, too.

Reply:
Yes good idea!